test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -217,5 +216,127 @@ public void TestWhiteningOldSavingAndLoading()
             }
             Done();
         }
+
+        [Fact]
+        public void LpNormWorkout()
+        {
```

Comment:
My newfound PR review line: - How do you feel about adding a sample for those two estimator :)?

Reply:
Can you give me an example of sample? Just want to know where I should add that stuff.

---
In reply to: [232162480](https://github.com/dotnet/machinelearning/pull/1582#discussion_r232162480) [](ancestors = 232162480)

Reply:
see the samples folder: /docs/samples/Microsoft.ML.Samples/Dynamic

Don't want to hold you up on that though. Can log an issue . 

---
In reply to: [233560783](https://github.com/dotnet/machinelearning/pull/1582#discussion_r233560783) [](ancestors = 233560783,232162480)